lib/util.js

```diff
@@ -371,6 +373,16 @@ function _exceptionWithHostPort(...args) {
   return new ExceptionWithHostPort(...args);
 }
 
+/**
+ * Parses the content of an `.env` file.
```

Comment:
```suggestion
 * Parses the content of an `.env` file.
```

Technically, it's not a `.env` file, right? It can be any utf8 string

Reply:
Yes that's correct

Reply:
```suggestion
 * Parses the content of a `.env` file.
```